.goreleaser.yaml

```
@@ -0,0 +1,40 @@
+# This is an example .goreleaser.yml file with some sensible defaults.
+# Make sure to check the documentation at https://goreleaser.com
+before:
+    hooks:
+        # You may remove this if you don't use go modules.
+        - go mod tidy
+builds:
+    - env:
+          - CGO_ENABLED=0
+      id: zbpack
+      binary: zbpack
```

CI: Our `zbpack` binary is placed at `cmd/zbpack`. So you may need to write the `main: ./cmd/zbpack` declaration in this build.

```suggestion
      main: ./cmd/zbpack
      binary: zbpack
```